src/components/HelFormFields/HelDateTimeField.js

```diff
@@ -143,14 +151,21 @@ class HelDateTimeField extends React.Component {
     shouldComponentUpdate(nextProps, nextState) {
         return true
     }
+    
+    resetFields = () => {
+        this.setState({
+            date: null,
+            time: null,
+        })
+    }
 
     render () {
         return (
             <div className="multi-field">
                 <div className="indented">
                     <label style={{position: 'relative'}}><FormattedMessage id={`${this.props.label}`} /> <ValidationPopover validationErrors={this.props.validationErrors} /></label>
-                    <HelDatePicker {...this.props.datePickerProps} ref="date" name={this.props.name} defaultValue={this.state.date} validations={[CONSTANTS.VALIDATION_RULES.IS_DATE]} placeholder="pp.kk.vvvv" onChange={this.onChange} onBlur={this.onBlur} label={<FormattedMessage id="date" />} />
-                    <HelTimePicker {...this.props.timePickerProps} ref="time" name={this.props.name} defaultValue={this.state.time} validations={[CONSTANTS.VALIDATION_RULES.IS_TIME]} placeholder="hh.mm" onChange={this.onChange} onBlur={this.onBlur} label={<FormattedMessage id="time" />} />
+                    <HelDatePicker {...this.props.datePickerProps} ref="date" name={this.props.name} defaultValue={this.state.date} validations={[CONSTANTS.VALIDATION_RULES.IS_DATE]} placeholder="pp.kk.vvvv" onChange={this.onChange} onBlur={this.onBlur} label={<FormattedMessage id="date" isClearable />} />
+                    <HelTimePicker {...this.props.timePickerProps} ref="time" name={this.props.name} defaultValue={this.state.time} validations={[CONSTANTS.VALIDATION_RULES.IS_TIME]} placeholder="hh.mm" onChange={this.onChange} onBlur={this.onBlur} label={<FormattedMessage id="time" isClearable />} />
```

Comment:
1. What does the `isClearable` prop do?
2. Could the props be stacked instead of inline to make the code more readable?

Reply:
Yeah, I don't see anything in `FormattedMessage` that would use this prop, where is it used?